hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiDeploymentBaseTest.java

```diff
@@ -947,6 +938,41 @@ public void badDeplomentActionFeedback() throws Exception {
 
     }
 
+    @Test
```

Comment:
I'd like to split these test up in two different ones because they actually testing different use-cases. One for missing the `result` in the feedback and one time the `finished` attribute. Then the test-case can have a better name e.g. `missingResultAttributeInFeedbackReturnsBadRequest`